pastepwn/analyzers/tests/creditcardanalyzer_test.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import unittest
+from unittest import mock
+
+from pastepwn.analyzers.creditcardanalyzer import CreditCardAnalyzer
+
+
+class TestCreditCardAnalyzer(unittest.TestCase):
+    def setUp(self):
+        self.analyzer = CreditCardAnalyzer(None)
+        self.paste = mock.Mock()
+
+    def test_match_positive(self):
+        """Test if positives are recognized
+        * CARDS IN ORDER * 
+        VISA
+        MASTERCARD
+        AMERICAN EXPRESS
+        DINERS CLUB
+        DISCOVER
+        JCB
+        """
+        self.paste.body = "4556316812657526"
+        self.assertTrue(self.analyzer.match(self.paste))
+        self.paste.body = "4556 3168 1265 7526"

```

Comment:
Good catch. The analyzer does not match credit card numbers with spaces in it yet. See [travis result](https://travis-ci.org/d-Rickyy-b/pastepwn/jobs/594094038)